genesyscloud/idp_adfs/resource_genesyscloud_idp_adfs.go

```diff
@@ -0,0 +1,145 @@
+package idp_adfs
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	resourceExporter "terraform-provider-genesyscloud/genesyscloud/resource_exporter"
+	"terraform-provider-genesyscloud/genesyscloud/util"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v129/platformclientv2"
+
+	"terraform-provider-genesyscloud/genesyscloud/consistency_checker"
+
+	"terraform-provider-genesyscloud/genesyscloud/util/constants"
+	"terraform-provider-genesyscloud/genesyscloud/util/lists"
+	"terraform-provider-genesyscloud/genesyscloud/util/resourcedata"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
+)
+
+/*
+The resource_genesyscloud_idp_adfs.go contains all of the methods that perform the core logic for a resource.
+*/
+
+// getAllAuthIdpAdfs retrieves all of the idp adfs via Terraform in the Genesys Cloud and is used for the exporter
+func getAllAuthIdpAdfss(ctx context.Context, clientConfig *platformclientv2.Configuration) (resourceExporter.ResourceIDMetaMap, diag.Diagnostics) {
+	proxy := newIdpAdfsProxy(clientConfig)
```

Comment:
this should be `getIdpAdfsProxy`. We should only really be calling the `newProxy` method from inside the proxy file